internal/infrastructure/storage/db/pg/migration/20230504102232_init.up.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE market (
+    name VARCHAR(50) NOT NULL PRIMARY KEY,
+    base_asset VARCHAR(64) NOT NULL,
+    quote_asset VARCHAR(64) NOT NULL,
+    base_asset_precision INTEGER,
+    quote_asset_precision INTEGER,
```

Comment:
These fields don't need a default but must not be null.

```suggestion
    base_asset_precision INTEGER NOT NULL,
    quote_asset_precision INTEGER NOT NULL,
```